documentation/doc.css

```diff
@@ -40,3 +42,19 @@ iframe {
   justify-content: center;
   flex-wrap: wrap;
 }
+
+.input-box-section {
+  display: flex;
+  justify-content: flex-end;
+  width: 50%;
+}
```

Comment:
spacing

Reply:
Somehow prettier did not work there.

Reply:
> Somehow prettier did not work there.

yeah I know same here it doesn't work sometime.